Packs/PhishLabs/Scripts/PhishLabsPopulateIndicators/PhishLabsPopulateIndicators.py

```diff
@@ -18,79 +18,88 @@ def indicator_type_finder(indicator_data: dict):
+
+    if isError(entry):
+        demisto.results('Failed getting the global feed from PhishLabs - {}'.format(entry['Contents']))
     else:
-        for indicator in feed:
-            indicator_type = indicator_type_finder(indicator)
-            indicator_value = indicator.get('value')
-            indicator_timestamp = None
-            if indicator.get('createdAt'):
-                indicator_timestamp = datetime.strptime(indicator['createdAt'], '%Y-%m-%dT%H:%M:%SZ')
-            if indicator_type == 'Attachment':
-                indicator_type = 'File MD5'
-                file_md5_attribute = list(filter(lambda f: f.get('name') == 'md5', indicator.get('attributes', [])))
-                indicator_value = file_md5_attribute[0].get('value') if file_md5_attribute else ''
-
-            demisto_indicator = {
-                'type': indicator_type_finder(indicator),
-                'value': indicator_value,
-                'source': 'PhishLabs',
-                'reputation': 'Bad',
-                'seenNow': 'true',
-                'comment': 'From PhishLabs Global Feed'
-            }
-
-            if indicator_timestamp:
-                demisto_indicator['sourceTimeStamp'] = datetime.strftime(indicator_timestamp, '%Y-%m-%dT%H:%M:%SZ')
-            indicator_res = demisto.executeCommand('createNewIndicator', demisto_indicator)
-
-            if isError(indicator_res[0]):
-                return_error('Error creating indicator - {}'.format(indicator_res[0]['Contents']))
-
-    demisto.results('Successfully populated indicators')
+        content = entry.get('Contents')
+        if not content or not isinstance(content, dict):
+            return_error('No indicators found')
+
+        feed = content.get('data', [])
+
+        if delete_false_positive:
+            false_positives = list(filter(lambda f: bool(strtobool(str(f.get('falsePositive', 'false')))) is True, feed))
+            for false_positive in false_positives:
+                delete_res = demisto.executeCommand('deleteIndicators',
+                                                    {'query': 'source:"PhishLabs" and value:"{}"'
+                                                        .format(false_positive.get('value')),
+                                                     'reason': 'Classified as false positive by PhishLabs'})
+                if isError(delete_res[0]):
+                    return_error('Error deleting PhishLabs indicators - {}'.format(delete_res[0]['Contents']))
+        else:
+            for indicator in feed:
+                indicator_type = indicator_type_finder(indicator)
+                indicator_value = indicator.get('value')
+                indicator_timestamp = None
+                if indicator.get('createdAt'):
+                    indicator_timestamp = datetime.strptime(indicator['createdAt'], '%Y-%m-%dT%H:%M:%SZ')
+                if indicator_type == 'Attachment':
```

Comment:
Hmm, kind of strange given this line here. maybe it's because of `indicator_type_finder`. Take a look and remove if needed.

Reply:
Ah I see the problem then - the indicator type is reflected in line 86 and I missed it. Also will change this to File